src/app/lab/lab.component.ts

```diff
@@ -27,95 +25,33 @@ import * as _ from 'lodash';
 })
 export class LabComponent implements OnInit, OnDestroy {
 
-  public apiCities: string = apiHost + "/api/city/?search=:keyword";
-  public climateModels: ClimateModel[] = [];
-  public scenarios: Scenario[] = [];
   public project: Project;
 
-  constructor(private chartService: ChartService,
-              private climateModelService: ClimateModelService,
-              private scenarioService: ScenarioService,
-              private projectService: ProjectService) {
+  constructor(private projectService: ProjectService,
+              private route: ActivatedRoute,
+              private router: Router) {
   }
 
   ngOnInit() {
-    // Initialize a project if it doesn't exist, otherwise just use an existing one
-    // TODO: This logic should be replaced with actual project handling code
-    //       once the feature exists
-    let projects = this.projectService.list();
-    if (projects.length) {
-      this.project = projects[0];
-    } else {
-      this.project = new Project({
-        id: 'test',
-        name: 'Test Project',
-        description: 'Placeholder for projects feature',
-        scenario: {name: defaultScenario, description: ''},
-        city: defaultCity
-      });
-      this.projectService.add(this.project);
-    }
-
-    this.getClimateModels();
-    this.getScenarios();
+    // Load existing project id or redirect to dashboard
+    this.route.params.subscribe(params => {
+      let id: string = params['id'];
+      if (id !== undefined) {
+        this.project = this.projectService.get(id);
```

Comment:
Probably also want to guard against the case where projectService.get is passed an invalid ID and doesn't find a matching project, and navigate back to dashboard in that case as well.